CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 ## UNRELEASED
 
+Improvements:
+
+* Support Consul namespaces [Enterprise feature] in all consul-k8s components
+* Create allow and deny lists of k8s namespaces for catalog sync and Connect inject
+* Significantly increases the test coverage of all components
+
```

Comment:
Missing:
* Connect Inject
  * Changes default Consul Docker image (`-consul-image`) to `consul:1.7.1`
  * Changes default Envoy Docker image (`-envoy-image`) to `envoyproxy/envoy-alpine:v1.13.0`